src/link/object.c

```diff
@@ -279,8 +279,7 @@ static void readPatch(FILE *file, struct Patch *patch, char const *fileName, cha
 	tryReadlong(patch->pcSectionID, file,
 		    "%s: Unable to read \"%s\"'s patch #%" PRIu32 "'s PC offset: %s",
 		    fileName, sectName, i);
-	patch->pcSection = patch->pcSectionID == (uint32_t)-1 ? NULL
-							      : fileSections[patch->pcSectionID];
+	patch->pcSection = NULL; /* set later by linkPatchToPCSect */
```

Comment:
If it's going to be filled in, why is this initialization needed? If it isn't, the comment should be enough.